lib/content-services/src/lib/search/components/search-panel/search-panel.component.scss

```diff
@@ -0,0 +1,3 @@
+.adf-search-panel {
```

Comment:
not needed

Reply:
I suggest removing this file and references to it, and adding only when we really need some new styles to be added